[2;35m2017-11-21 00:17:35,852 - WARN  [Thread-13:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :14, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[7;31m2017-11-21 00:22:08,726 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x18bc8094, /127.0.0.1:59791 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 00:22:08,750 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xf7f9f716, /127.0.0.1:59790 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 00:22:08,760 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x9a6f7893, /127.0.0.1:59789 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-11-21 00:23:04,847 - INFO  [main:ServerBootstrap@99] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-11-21 00:23:05,269 - INFO  [main:ServerBootstrap@125] - Configuration Server bootstrap params.......
[m[2;32m2017-11-21 00:23:05,269 - INFO  [main:ServerBootstrap@126] - Server protocol:	socket
[m[2;32m2017-11-21 00:23:05,270 - INFO  [main:ServerBootstrap@127] - Server port:	9912
[m[2;35m2017-11-21 00:23:05,552 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-11-21 00:23:05,572 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-11-21 00:23:05,582 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-11-21 00:23:05,592 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-11-21 00:23:05,593 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-11-21 00:23:05,602 - INFO  [main:ServerBootstrap@150] - Application PID:	24700
[m[2;35m2017-11-21 00:23:11,871 - WARN  [Thread-22:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :14, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[7;31m2017-11-21 00:27:47,740 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xf8fefa83, /127.0.0.1:60113 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 00:27:47,748 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x1501005a, /127.0.0.1:60112 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 00:27:47,743 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x648908bf, /127.0.0.1:60111 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-11-21 10:20:24,764 - INFO  [main:ServerBootstrap@99] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-11-21 10:20:25,299 - INFO  [main:ServerBootstrap@125] - Configuration Server bootstrap params.......
[m[2;32m2017-11-21 10:20:25,299 - INFO  [main:ServerBootstrap@126] - Server protocol:	socket
[m[2;32m2017-11-21 10:20:25,299 - INFO  [main:ServerBootstrap@127] - Server port:	9912
[m[2;35m2017-11-21 10:20:25,496 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-11-21 10:20:25,514 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-11-21 10:20:25,524 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-11-21 10:20:25,531 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-11-21 10:20:25,531 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-11-21 10:20:25,541 - INFO  [main:ServerBootstrap@150] - Application PID:	5868
[m[2;35m2017-11-21 10:20:30,644 - WARN  [Thread-92:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :35, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[7;31m2017-11-21 10:41:28,414 - ERROR [Xmemcached-Reactor-2:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2017-11-21 10:41:28,433 - WARN  [Xmemcached-Reactor-2:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[2;35m2017-11-21 10:41:30,436 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[2;35m2017-11-21 10:41:30,437 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;35m2017-11-21 10:42:04,633 - WARN  [Thread-152:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :20, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[2;35m2017-11-21 10:43:30,384 - WARN  [Thread-32:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :4, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[2;35m2017-11-21 10:43:39,220 - WARN  [Thread-87:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :15, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOtherOrderReqInfo
[m[2;35m2017-11-21 10:50:52,739 - WARN  [Thread-153:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :3, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[2;35m2017-11-21 11:04:50,646 - WARN  [Thread-2:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :5, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[2;35m2017-11-21 11:07:15,592 - WARN  [Thread-12:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :6, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[2;35m2017-11-21 12:22:43,822 - WARN  [Thread-123:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :2, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[7;31m2017-11-21 12:23:12,518 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x2f009bd5, /127.0.0.1:62087 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 12:23:12,518 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x4af52e73, /127.0.0.1:62088 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 12:23:12,524 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xecb68722, /127.0.0.1:62089 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-11-21 12:34:05,035 - INFO  [main:ServerBootstrap@99] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-11-21 12:34:05,474 - INFO  [main:ServerBootstrap@125] - Configuration Server bootstrap params.......
[m[2;32m2017-11-21 12:34:05,474 - INFO  [main:ServerBootstrap@126] - Server protocol:	socket
[m[2;32m2017-11-21 12:34:05,474 - INFO  [main:ServerBootstrap@127] - Server port:	9912
[m[2;35m2017-11-21 12:34:05,727 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-11-21 12:34:05,746 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-11-21 12:34:05,755 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-11-21 12:34:05,762 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-11-21 12:34:05,765 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-11-21 12:34:05,781 - INFO  [main:ServerBootstrap@150] - Application PID:	25068
[m[2;35m2017-11-21 12:34:08,735 - WARN  [Thread-132:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :48, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[2;35m2017-11-21 12:35:29,605 - WARN  [Thread-133:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :464, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOtherOrderReqInfo
[m[2;35m2017-11-21 13:55:16,236 - WARN  [Thread-62:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :4, action:com.klw.fastfun.pay.common.action.CommonAction, method:getPollOtherPayCodes
[m[7;31m2017-11-21 14:32:01,872 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x405cecb7, /127.0.0.1:65493 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 14:32:01,872 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x9dcb5f39, /127.0.0.1:65494 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-11-21 14:32:01,872 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x0b5baf25, /127.0.0.1:65492 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
